include/GoogleSync/GoogleSync.php

```diff
@@ -1079,26 +1231,26 @@ public function doSync($id)
 
             switch ($dowhat) {
                 case "push":
-                    $GLOBALS['log']->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Pushing Event: ' . $gevent->getSummary());
+                    $this->logger->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Pushing Event: ' . $gevent->getSummary());
                     $this->pushEvent($meeting, $gevent);
                     break;
                 case "pull":
-                    $GLOBALS['log']->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Pulling Event: ' . $gevent->getSummary());
+                    $this->logger->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Pulling Event: ' . $gevent->getSummary());
                     $this->pullEvent($gevent, $meeting);
                     break;
                 case "skip":
-                    $GLOBALS['log']->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Skipping Event: ' . $gevent->getSummary());
+                    $this->logger->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Skipping Event: ' . $gevent->getSummary());
                     break;
                 case "push_delete":
-                    $GLOBALS['log']->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Deleting Event: ' . $meeting->name);
+                    $this->logger->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Deleting Event: ' . $meeting->name);
                     $this->delEvent($gevent, $meeting->id);
                     break;
                 case "pull_delete":
-                    $GLOBALS['log']->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Deleting Meeting: ' . $meeting->name);
+                    $this->logger->info(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Deleting Meeting: ' . $meeting->name);
                     $this->delMeeting($meeting);
```

Comment:
delMeeting() couls returns false on failure. Please handle the error.

Reply:
return is now passed to caller